monolith/src/main/java/com/society/server/web/CommentController.java

```diff
@@ -16,31 +17,36 @@
 public class CommentController {
 
     private final CommentService commentService;
+    private final PostService postService;
 
-    public CommentController(CommentService commentService) {
+    public CommentController(CommentService commentService, PostService postService) {
         this.commentService = commentService;
+        this.postService = postService;
     }
 
-    @PostMapping()
+
+    @PostMapping("/comment")
     public ResponseEntity<CommentDTO> createComment(@RequestHeader("X-username") String username,
+                                                    @RequestParam("postId") Long id,
                                                     @Valid @RequestBody CreateCommentDTO createCommentDTO) {
-        CommentDTO commentDTO = commentService.createComment(username, createCommentDTO);
-
+        CommentDTO commentDTO = commentService.createComment(id, username, createCommentDTO);
         return ResponseEntity
                 .status(HttpStatus.CREATED)
                 .body(commentDTO);
     }
 
-    @DeleteMapping("/commentId")
-    public ResponseEntity<CommentDTO> deleteComment(@PathVariable("commentId") Long id) {
+    //Todo: Fix deleteComment method
+    @DeleteMapping("/{commentId}")
+    public ResponseEntity<CommentDTO> deleteComment(@RequestHeader("X-username") String username,
+                                                    @PathVariable("commentId") Long id) {
         commentService.deleteComment(id);
-
         return ResponseEntity
                 .status(HttpStatus.OK)
                 .build();
     }
 
-    @PatchMapping("/update/commentId")
+    //Todo: Fix updateComment method
+    @PatchMapping("/{commentId}")
```

Comment:
Why not @PutMapping. And I think the path needs to be `/comment/{commentId}` like the createComment to not mess up with the path vars

Reply:
This method is not ready, will be fixed in next pull request.